git stater pack

```diff
@@ -0,0 +1,13 @@
+### git stater pack
+-------------------------------------------------------------------------
+1. Create a Repo on github
+2. Copy the https link of the Repo
+3. Go to the terminal and install git
+4. use command
+    "git clone <enter HTTPS link without any bracket>" 
+5. Coppy your project in the foder created after the command in step '4'
```

Comment:
@sidsau97 a little help:
`Copy your project in the folder created after the command in step`